go.mod

```diff
@@ -87,7 +86,7 @@ require (
 	github.com/moby/sys/mountinfo v0.4.0 // indirect
 	github.com/moby/term v0.0.0-20201216013528-df9cb8a40635 // indirect
 	github.com/morikuni/aec v1.0.0 // indirect
-	github.com/nxadm/tail v1.4.8 // indirect
+	github.com/onsi/ginkgo/v2 v2.1.3
```

Comment:
Since ginkgo is a direct dependency, shouldn't it be in the upper 'require' section?

Reply:
updated.